peas/weather.py

```diff
@@ -103,8 +103,7 @@ class AAGCloudSensor(object):
 
     def __init__(self, serial_address=None, use_mongo=True):
         self.config = load_config(config_files='peas')
-        self.logger = logging.getLogger('aag-cloudsensor')
-        self.logger.setLevel(logging.INFO)
+        self.logger = get_root_logger()
```

Comment:
As commented above, we could pass `aag-cloudsensor` as the profile here and then change the log output format to include this `name`, which would be easily filterable.

This file does output a lot of debug information each time it queries the weather, however, which isn't a bad thing but mostly ends up being noise.

Reply:
I looked at the log file, and I agree, lots of ~~noise~~ details. :wink: